test/code_block_fenced.md

```diff
@@ -9,9 +9,16 @@ And here is more text
 This is a code block that won't trigger.
 ```
 
+```text
+This is code
+ 
+with an empty line.
+```
+
 But we'll do another:
 
     And this {MD046}
     will.
 
 Final text is here
+{MD009:14}
```

Comment:
Can you put this at the end of line 13 or 15 so it’s closer?

Reply:
Aha, no idea you could do that. Cool, will do.